src/routes/TraderForm/Form/TransactionDetails/index.tsx

```diff
@@ -17,6 +19,16 @@ export const TransactionDetails: React.FunctionComponent<TransactionDetailsProps
   const infoPostSwap = useAppSelector(selectInfoPostSwap);
   const token = useAppSelector(selectAMMTokenFormatted);
   const submitButtonInfo = useAppSelector(selectSubmitButtonInfo);
+  const slippage = useAppSelector(selectSlippage);
+
+  let fee = '--';
+  if (infoPostSwap.status === 'success') {
+    if (infoPostSwap.value.fee < 1) {
+      fee = formatNumber(infoPostSwap.value.fee, 0, SwapFormNumberLimits.decimalLimit);
```

Comment:
wait so we format up to 6 digits?

Reply:
In the case fee is super small, you cannot really understand what is the value. This point was raised during QA